### ISSUE TITLE
NumPy: the absolute basics for beginners - as a jupyter notebook in Swedish.

### ISSUE THREAD

**Issue by AakeJon49** [numpy/numpy-tutorials#204]
I have translated and made a jupyter note book of NumPy: the absolute basics for beginners.
I have made it for my own hard to digest new knowledge acquisition, maybe you have use of it?

**Comment by rossbar** [comment]
Hi @AakeJon49 , thanks for the ping. The landscape can be a bit confusing, so I'll try to distill the current state re: docs + translations:
 - Currently, the only resource in the NumPy organization that supports translations is the official organization website at https://numpy.org  [(source)](https://github.com/numpy/numpy.org). The translations are managed via [crowdin](https://crowdin.com/project/numpyorg).
 - Neither the official documentation (available at https://numpy.org/doc) nor the tutorials site (https://numpy.org/numpy-tutorials - i.e. this repo) support translations.
 - The absolute beginners tutorial is part of the official numpy documentation, for which translations are not supported.

So in summary - it's not currently possible to host/link to translations of the absolute beginner's tutorial. Thanks for pinging about sharing though, it's great that you have done this translation and are willing to share back!

I will close the issue, but if there are further inquiries please don't hesitate to start a discussion on a better-suited forum such as the mailing list!
